src/classes/CallableApi.cls

```diff
@@ -55,15 +55,16 @@ global with sharing class CallableApi implements System.Callable {
         switch on (cleanActionText) {
             when 'tdtm.disablealltriggers' {
                 TDTM_Config_API.disableAllTriggers();
-            }
-            when else {
+            } when 'opp.mapstagetostate' {
```

Comment:
Can you add a blank line between the two when's to make this a bit more readable as it grows?
```
when 'tdtm.disablealltriggers' {
    // do something

} when 'opp.mapstagetostate' {
   // do something else

} when else {
   // throw exception

}
```